feature-group-definitions/media-capture.dist.yml

```diff
@@ -0,0 +1,46 @@
+# Generated from: media-capture.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Media capture
+description: TODO
+spec: https://w3c.github.io/mediacapture-main/
+caniuse: stream
+# Override is needed to fix the Edge version:
+# https://github.com/mdn/browser-compat-data/pull/22913
+#
+# Firefox support for `enumerateDevices()` also needs an override since lack of
+# output device (a separate feature) is treated as partial implementation in
+# BCD.
+status:
+  baseline: high
+  baseline_low_date: 2017-09-19
+  baseline_high_date: 2020-03-19
+  support:
+    chrome: "53"
+    chrome_android: "53"
+    edge: "12"
+    firefox: "39"
```

Comment:
The Chrome, Edge and Safari versions in https://caniuse.com/stream match here, but Firefox doesn't, caniuse says 36. That comes down to me including `enumerateDevices()` in this feature, which I think is reasonable, since `enumerateDevices()` on its own doesn't amount to much, it's in support of either media capture or audio device output.